src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCounter = 0;
+
+  return (x) => {
+    return (y) => {
+      callCounter++;
+
+      if (callCounter <= 3) {
+        return x + y;
+      } else if (callCounter % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return x + y;
+      }
```

Comment:
on your variant, you return x + y two times. When you do similar logic, try to combine conditions

```suggestion
      if (count <= 3 || count % 2 !== 0) {
        return x + y;
      }
      
      return 'Bzzz... Error!';
```